src/Logging/Logging.Testing/src/LoggedTest/LoggedTestBase.cs

```diff
@@ -63,12 +64,17 @@ public virtual void Initialize(TestContext context, MethodInfo methodInfo, objec
                 var logLevelAttribute = methodInfo.GetCustomAttribute<LogLevelAttribute>()
                                         ?? methodInfo.DeclaringType.GetCustomAttribute<LogLevelAttribute>()
                                         ?? methodInfo.DeclaringType.Assembly.GetCustomAttribute<LogLevelAttribute>();
+                var flakyAttribute = methodInfo.GetCustomAttribute<FlakyAttribute>()
+                                        ?? methodInfo.DeclaringType.GetCustomAttribute<FlakyAttribute>()
+                                        ?? methodInfo.DeclaringType.Assembly.GetCustomAttribute<FlakyAttribute>();
```

Comment:
Curious: Doesn't the generic method have an overload supporting `inherit: true`?